CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ endif(IS_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/.git)
 find_package(Threads REQUIRED)
 set(THREAD_LIBRARIES ${CMAKE_THREAD_LIBS_INIT})
 
-find_package(Boost 1.39.0 REQUIRED COMPONENTS program_options filesystem system )
+find_package(Boost 1.57.0 REQUIRED COMPONENTS program_options filesystem system )
```

Comment:
Does this mean that Boost 1.57.0 is the minimum required version?

Reply:
yes. the rest of votca has that already